temp.py

```diff
@@ -0,0 +1,2 @@
+import pdb
```

Comment:
Avoid using the `pdb` (Python Debugger) in committed code. If debugging is necessary, consider using logging or other debugging techniques that do not require modifying the codebase.